ipn/ipnlocal/c2n.go

```diff
@@ -319,6 +319,19 @@ func findCmdTailscale() (string, error) {
 			return ts, nil
 		}
 		return "", errors.New("tailscale.exe not found in expected place")
+	case "darwin":
+		if strings.HasSuffix(self, "/Contents/MacOS/Tailscale") {
+			return self, nil
+		}
+		out, err := exec.Command("mdfind", "kMDItemCFBundleIdentifier == 'io.tailscale.ipn.macsys'").CombinedOutput()
```

Comment:
This can return more than one path, one on each line, if multiple instances are found

Reply:
See https://stackoverflow.com/a/10063710

Reply:
Something evil is happening with my local build, I get ` c2n: POST /update failed: failed to find cmd/tailscale binary: signal: trace/BPT trap` when testing the latest PR version.

The check for `self` suffix fails, probably because this is running `/Library/SystemExtensions/56E2CCF7-D2C1-4F31-AA18-32ADE1F5F116/io.tailscale.ipn.macsys.network-extension.systemextension/Contents/MacOS/io.tailscale.ipn.macsys.network-extension` and not the regular app binary.

If I do the `mdfind` check, it hits the second error condition (`strings.TrimSpace(out)` being empty).
If I try to print the raw output of `mdfind` (as in the latest PR version), I get that `trace/BPT trap` ☠️ 

Going to debug this more after lunch.